packages/serverless-logic-web-tools/src/samples/SampleCard.tsx

```diff
@@ -148,6 +129,18 @@ export function SampleCard(props: { sample: Sample; cover: string | undefined })
                 </Text>
               </Tooltip>
             </CardBody>
+            <CardFooter>
+              <Link
+                to={{
+                  pathname: routes.sampleShowcase.path({}),
+                  search: routes.sampleShowcase.queryString({ sampleId: props.sample.sampleId }),
+                }}
+              >
+                <Button variant={ButtonVariant.tertiary} ouiaId={props.sample.sampleId + `-try-swf-sample-button`}>
```

Comment:
```suggestion
                <Button variant={ButtonVariant.tertiary} ouiaId={props.sample.sampleId + `-try-sample-button`}>
```
Not all samples are swf.